plugin/pkg/admission/resourcequota/config.go

```diff
@@ -37,19 +37,35 @@ func init() {
 	install.Install(scheme)
 }
 
+func defaultLimitedResources() []resourcequotaapi.LimitedResource {
+	return []resourcequotaapi.LimitedResource{
+		{
+			Resource: "pods",
+			MatchScopes: []corev1.ScopedResourceSelectorRequirement{
+				{
+					ScopeName: corev1.ResourceQuotaScopePriorityClass,
+					Operator:  corev1.ScopeSelectorOpIn,
+					Values:    []string{"system-cluster-critical", "system-node-critical"},
+				},
+			},
+		},
+	}
+}
+
+func DefaultAdmissionConfig() *resourcequotaapi.Configuration {
```

Comment:
I want to use this function in test cases, so exposing this as a public function.